lib/sql/escape.go

```diff
@@ -25,7 +25,7 @@ func NeedsEscaping(name string, destKind constants.DestinationKind) bool {
 	if destKind == constants.Redshift {
 		reservedKeywords = constants.RedshiftReservedKeywords
 	} else if destKind == constants.MSSQL {
-		reservedKeywords = constants.MSSQLReservedKeywords
+		return !strings.HasPrefix(name, constants.ArtiePrefix)
```

Comment:
Why not just escape all?

Reply:
If we escape all we get test that say `artie delete flag doesn't exist`.

Reply:
I'm guessing there must be some places where we escape the column names and then check if the column names contain a specific value.

Reply:
That's a guardrail in `dml/merge`

Reply:
The comparison is running `"__artie_deleted` == __artie_deleted` which will always equate to false